### PR TITLE
Updating date and other fixes to the distribution packaging topic

### PR DESCRIPTION
**Pull request by mairaw**
Updating the date, fixing typos and things that were not following our style guide conventions. I'm also changing the author to @tmds since he made significant changes here.

### CODE REVIEW

File: docs/core/build/distribution-packaging.md [hunk 1]
```diff
@@ -48,34 +44,34 @@ When installed, .NET Core consists of several components that are layed out as f
...
+- (2) **host/fxr/\<fxr version>** contains the framework resolution logic used by the host. The host uses the latest hostfxr that is installed. The hostfxr is responsible for selecting the appropriate runtime when executing a .NET Core application. For example, an application built for .NET Core 2.0.0 uses the 2.0.5 runtime when it's available. Similarly, hostfxr selects the appropriate SDK during development.
 
-- (3) **sdk/\<sdk version>** The SDK (also known as "the tooling") is a set of managed tools that can be used to write and build .NET Core libraries and applications. The SDK includes the CLI, the Roslyn compiler, MSBuild, and associated build tasks and targets, NuGet, new project templates, etc.
+- (3) **sdk/\<sdk version>** The SDK (also known as "the tooling") is a set of managed tools that can be used to write and build .NET Core libraries and applications. The SDK includes the .NET Core Command-line interface (CLI), the Roslyn compiler, MSBuild, and associated build tasks and targets, NuGet, new project templates, etc.
```
Comment (BillWagner) [inline docs/core/build/distribution-packaging.md]: Not related to this PR, but I think "the roslyn compiler" is not preferred, and saying "the managed languages compilers" would be better. (That would include C#, VB.NET, and F#).

**Review by BillWagner — APPROVED** [review]
Overall, this looks good.  I had one minor comment.

You can ship it when ready, @mairaw

**Comment by mairaw** [comment]
Thanks Bill. There were some extra feedback on the last PR for this that I still need to address as well.

**Comment by tmds** [comment]
@mairaw @tannergooding thank you for improving the text!

**Pull request merged**